<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>新用户注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>住址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电子邮箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUpBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="escBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>用户名注册后无法更改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="adressEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telNumEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>escBtn</sender>
   <signal>clicked()</signal>
   <receiver>SignUp</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
